frontend/src/components/filters/MultiSelect.tsx

```diff
@@ -50,21 +101,71 @@ export const MultiSelect = ({
           />
           <MenuButton />
         </MenuHandler>
-        <MenuList className={field.name}>
-          {field.options?.map((option: any) => (
-            <MenuItem key={option}>
-              <label className={classes.menu.listItem} htmlFor={option.label}>
-                <Checkbox
-                  onChange={onChange}
-                  checked={values?.includes(option)}
-                  name={field.name}
-                  value={option}
-                />
-                {option}
-              </label>
-            </MenuItem>
-          ))}
-        </MenuList>
+        {field.groupedOptions ? (
```

Comment:
combined MultiSelectGroup with MultiSelect because the logic for setting the values is shared, and only the MenuList view is different so this is now set conditionally